docs/getting-started.md

```diff
@@ -22,9 +22,14 @@ make docker-build
 MySQL setup
 -----------
 
-After you have MySQL up and running, create a new empty database (/schema) for
+After you have MySQL up and running, create new empty database (/schema) & user for
 `a12n-server`.
 
+```sh
+mysql> CREATE DATABASE db_name;
+mysql> CREATE USER 'username'@'localhost' IDENTIFIED BY 'password';
```

Comment:
Might be a good idea to suggest `a12nserver` for both db and user, so it matches the defaults. Also mention that 'password' should be replaced by your own password.

Also drop the '@localhost` portion, and move up the GRANTS to the same section. No need to do this separate.